pom.xml

```diff
@@ -8,6 +8,14 @@
     <version>1.0-SNAPSHOT</version>
 
     <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>central</id>
+            <name>Central Repository</name>
+            <url>https://repo.maven.apache.org/maven2</url>
+        </repository>
```

Comment:
I don't think we need to add an additional repository as the `dp-logging` library is hosted by jitpack?

Reply:
ok, I will try to build and run the service without Central Repository

Reply:
validated and pushed a new commit without this dependency